dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.tsx

```diff
@@ -221,8 +223,13 @@ export const ConfigPanel = ({ visualizations, setCurVisId, callback, changeIsVal
   };
 
   const memorizedVisualizationTypes = useMemo(() => {
+    let visDefinition = {}
     return ENABLED_VIS_TYPES.map((vs: string) => {
-      const visDefinition = getVisType(vs);
+      if (vs === visChartTypes.Line || vs === visChartTypes.Scatter) {
+        visDefinition = vs === visChartTypes.Line ? getVisType(vs, { type: visChartTypes.Line }) : getVisType(vs, { type: visChartTypes.Scatter });
```

Comment:
I think, this can be done by the following. No need for an additional if condition.
```JS
visDefinition = getVisType(vs, { type: vs });
```